Open Judge System/Workers/OJS.Workers.Compilers/CPlusPlusZipCompiler.cs

```diff
@@ -39,25 +39,22 @@ public override string RenameInputFile(string inputFile)
         public override string BuildCompilerArguments(string inputFile, string outputFile, string additionalArguments)
         {
             var arguments = new StringBuilder();
-            // Output file argument
+
             arguments.Append($"-o \"{outputFile}\"");
             arguments.Append(' ');
 
-            // Additional compiler arguments
             arguments.Append(additionalArguments);
             arguments.Append(' ');
             FileHelpers.UnzipFile(inputFile, this.workingDirectory);
 
-            // Input files arguments
             var filesToCompile = Directory.EnumerateFiles(
```

Comment:
IMO .EnumeratFiles on new line; one tab; aligned with .Where below; other stuff one more tab to the right